content/20.treatments.md

```diff
@@ -132,7 +132,18 @@ A list of current studies and their results, using carefully the information req
 
 Summarize the state of the neutralizing antibody approach.
 
-#### Vaccines
+### Antiviral Approach: Remdesivir
+Remdesivir (GS-5734) was developed by Gilead Sciences to treat of Ebola and was briefly tested in the clinic.  It does not have any FDA approved use.  There was some evidence of effectiveness, but it was not as effective as two antibody preparations so it was not pursued. 
+https://www.sciencemag.org/news/2019/08/finally-some-good-news-about-ebola-two-new-treatments-dramatically-lower-death-rate
+   
+### Anticipated Mechanism
+Remdesivir is metabolized to GS-441524, an adenosine analog that inhibits a broad range of polymerases and then evades exonuclease repair causing chain termination (https://www.jbc.org/content/early/2020/02/24/jbc.AC120.013056.abstract)(https://mbio.asm.org/content/9/2/e00221-18).   Although it was developed against Ebola, it also inhibits the MERS CoV and SARS CoV polymerase and inhibits corona virus replication in cell culture assays with submicromolar IC50s (https://stm.sciencemag.org/content/9/396/eaal36530.  It also inhibits CoV-19 showing synergy with chloroquine.
+### Current Evidence
+Remdesivir was used on four Covid-19 patients under compassionate use guidelines. All were in late stages of Covid-19 infection and the results have been mixed.  More recently Remdesivir entered controlled clinical trials.  As of March 2020, there are six clinical trials underway to treat Covid-19 patients at both early and late stages of infection and in combinations with other drugs (https://www.nature.com/articles/s41422-020-0282-0). (NCT04292730, NCT04292899, NCT04280705, 2020-000936-23, NCT04252664, NCT04257656). 
+### Summary
+Remdesivir is a major drug candidate since it attacks the virus with high potency and known mechanism.  Moreover, one of the most successful therapies for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.  Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including HIV and Herpes.  Note that the HIV and Herpes polymerases are a reverse transcriptase and a DNA polymerase respectively, whereas Covid-19 encodes an RNA dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against Covid-19.    In clinical use, polymerase inhibitors show short term benefits for HIV patients but for long term benefits they must be part of combination regimens. They are typically combined with protease inhibitors, integrase inhibitors and even other polymerase inhibitors.   
```

Comment:
```suggestion
Remdesivir is a major drug candidate since it attacks the virus with high potency and known mechanism.  
Moreover, one of the most successful therapies for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.  
Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including HIV and Herpes.  
Note that the HIV and Herpes polymerases are a reverse transcriptase and a DNA polymerase respectively, whereas SARS-CoV-2 encodes an RNA dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against SARS-CoV-2.    
In clinical use, polymerase inhibitors show short term benefits for HIV patients but for long term benefits they must be part of combination regimens. 
They are typically combined with protease inhibitors, integrase inhibitors and even other polymerase inhibitors.   
```